com.georgefb.haruna.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.georgefb.haruna
+runtime: org.kde.Platform
+runtime-version: 5.14
+sdk: org.kde.Sdk
+command: haruna
+finish-args: 
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=ca.desrt.dconf
+  - --socket=pulseaudio
+  - --env=LC_NUMERIC=C
+
+modules:
+  - name: haruna
+    buildsystem: cmake-ninja
+    sources:
+     - type: git
+       url: https://github.com/g-fb/haruna
+       tag: 0.1.0
+    modules:
+      - kfilemetadata.yml
```

Comment:
Don't split the modules just for the sake of splitting them.

Reply:
Can I keep libmpv split? When is it ok to split them?

Reply:
Unless if it's generated automatically I would say.